docs/reference/health/health.asciidoc

```diff
@@ -260,6 +260,123 @@ for health status set `explain` to `false` to disable the more expensive analysi
 =====
 ====
 
+[role="child_attributes"]
+[[health-api-response-details]]
+==== Indicator Details
+
+Each health indicator in the health api returns a set of details that further explains the state of the system. The
+details have contents and a structure that is unique to each indicator.
+
+[[health-api-response-details-master-is-stable]]
+===== master_is_stable
+
+`current_master`::
+    (object) Information about the currently elected master.
++
+.Properties of `current_master`
+[%collapsible%open]
+====
+`node_id`::
+    (string) The node id of the currently elected master, or null if no master is elected.
+
+`name`::
+    (string) The node name of the currently elected master, or null if no master is elected.
+====
+
+`recent_masters`::
+    (Optional, array) A list of nodes that have been elected or replaced as master in a recent
+    time window (based on `master_history.max_age`). This field is present if the master
```

Comment:
```suggestion
    time window. This field is present if the master
```

I don't think we documented `master_history.max_age` yet. I think we should do that separately? And then mention it back here